### DISCUSSION TITLE
Can we add tags to the document right after saving it?

### DISCUSSION THREAD

**Discussion by GrayLanterns** [omnivore-app/omnivore#3699] [Q&A]
It's like a pop-up window which, alongside telling you that your article is saved, gives you an option to add relevant tags to your document for easier access.

A similar app that has the said feature I'm referring to: Pocket.

I am new to the community so I'm not sure if it has the feature I'm looking for. If not, please take this as feedback for future development.

Thank you for keeping this app open source.
Rooting for you devs. 🙌🏼

**Comment by jacksonh** [comment]
hey @GrayLanterns I assume you are on Android, right? We have this for our browser extension and on iOS but still need to figure out the UX on Android

**Comment by GrayLanterns** [comment]
Yes, mine is Android. Thanks for confirming it. The feature seems to be of utmost importance to me as I save a bunch of stuff from social media. Hope you can release it soon. My best wishes to you and the co dev team.

I have another query related with the contents of the saved document not showing in the app:

As you can see in the screenshot, when we save something from Twitter (X), it doesn't show the contents of the saved document in the app itself. Is it due to the API restrictions? Cos I face similar issues on Pocket as well.
![Screenshot_20240320_145157_Omnivore](https://github.com/omnivore-app/omnivore/assets/83039148/0d8b153d-040e-4a13-a538-d3f4522738fc)
